modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/components/RuleEditor/RuleEditor.soy

```diff
@@ -473,10 +473,11 @@
 					</div>
 
 					<div action-type-index="{$index}" class="action-type form-group-item">
-						{delcall PageRenderer.RegisterFieldType variant="'select'"}
+						{call ReactFieldAdapter.render}
```

Comment:
I'm still investigating but this doesn't seem to work very well in RuleEditor, some values ​​don't seem to activate other fields when selected. I'm guessing that there is a problem with the Metal adapter when the values ​​are boolean, it seems to have returned this problem here.

Reply:
But with `{call ReactFieldAdapter.render}` it works?

Reply:
I managed to fix this problem! Actually it was not a problem in the Adapter but because the file Type was sometimes empty and broke silently. I'll send the fix, disregard!

Reply:
> But with {call ReactFieldAdapter.render} it works?

It works but we have to pass the `fieldType` as a parameter.